docs/topics/contributing-before-you-start.adoc

```diff
@@ -37,14 +37,17 @@ To install Meld on Fedora, CentOS, or RHEL, run the following command (in Fedora
 
 To contribute, you must have a link:https://github.com/join[GitHub account] with GPG configured. To configure GPG:
 
-WARNING: On RHEL, CentOS, or Fedora, use the `gpg2` binary everywhere instead of `gpg`. On MacOS, use `gpg`.
+WARNING: On RHEL, CentOS, or Fedora, use the `gpg2` binary instead of `gpg`. On MacOS, use the `gpg` binary. 
 
 .Configuring GPG
+
+The command options you use to configure `gpg2` or `gpg` are the same for both binaries.
+
 . link:https://help.github.com/articles/generating-a-new-gpg-key/[Generate a new GPG key].
 . link:https://help.github.com/articles/adding-a-new-gpg-key-to-your-github-account[Add the GPG key to your GitHub account].
 . link:https://help.github.com/articles/telling-git-about-your-gpg-key/[Configure GPG with git] on your machine.
 +
-NOTE: On RHEL, CentOS, or Fedora, ensure to also add `gpg2` as the signing program according to the linked instructions.
+NOTE: Add `gpg2` or `gpg` as the signing program according to the linked instructions.
```

Comment:
Please make this a step rather than a note.
Consider converting the following 2 steps into sub-steps, since they both elaborate on how to add `gpg`/`gpg2` as a signing application in git.
Also, you seem to mention linked instructions, but the note is followed by a steps telling you how to set up your commit singing. Please consider removing that reference, because it sounds a little ambiguous.

Reply:
I agree with @inoxx03's comments.

Reply:
Thanks @inoxx03. I removed the note because it repeats the information in step 5. I also improved the wording for step 3 so that it more closely matches the linked procedures. Let me know what you think!

Reply:
Thank you for the updates @huntley, IMHO the section reads better this way:)